catalogue/webapp/components/WorkDetails/WorkDetails.js

```diff
@@ -84,15 +92,28 @@ const WorkDetails = ({
   });
 
   const WorkDetailsSections = [];
-  if (iiifImageLocationUrl) {
+  const allDownloadOptions = [
+    ...downloadOptions,
+    ...iiifPresentationDownloadOptions,
+  ];
+
+  useEffect(() => {
+    const iiifPresentationDownloadOptions =
+      Object.keys(iiifPresentationManifest).length !== 0
+        ? getDownloadOptionsFromManifest(iiifPresentationManifest)
+        : [];
+    setIIIFPresentationDownloadOptions(iiifPresentationDownloadOptions);
+  }, [iiifPresentationManifest]);
```

Comment:
I'd like to see how many times this effect get's run, we need to get some profiling in.

Reply:
before this is merged?

Reply:
Na, but I want profiling to be added pretty high priority, I'll talk to Jenn.